src/main/java/Client/ConsoleClientLauncher.java

```diff
@@ -91,8 +105,9 @@ private static void ConsoleMenue(){
                 "6: Initialize Model",
                 "7: Add Folder to ModelicaPath",
                 "8: Show ModelicaPath",
-                "9 : Exit",
-                "10 : Complete"
+                "9 : Exit - Shutdown Server",
+                "10 : Exit - Disconnect",
```

Comment:
What do you think about giving the ExitCommands the Numbers 30 and 31?
this would give us more space for future enhancements and still leave exits at the end of the menu